cicecore/cicedynB/dynamics/ice_transport_remap.F90

```diff
@@ -832,15 +826,7 @@ subroutine horizontal_remap (dt,             ntrace,   &
                              mm   (:,:,0,iblk))
 
          if (l_stop) then
-            this_block = get_block(blocks_ice(iblk),iblk)
-            write (nu_diag,*) 'istep1, my_task, iblk, cat =',     &
-                               istep1, my_task, iblk, '0'
```

Comment:
here also we loose this `cat = 0` bit of info, but I'm not sure if it was really useful...

Reply:
Agree, I don't think this adds much so will not re-add.